### DISCUSSION TITLE
why i lost my archive?

### DISCUSSION THREAD

**Discussion by nebulas16** [community/community#124618] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I play single game.I did not delete anything，but i have two game files,yesterday i clear the game ,today i use another game files because mistake.
now i can see my old archive in load game,but when i start a new game ,i only see the initial pokemon,how can i get it back?
![%QDY 6F O63Q}$F)K{S NDQ](https://github.com/community/community/assets/170230394/96cea3b8-7b0c-48a0-a93a-3f992c6d3c27)
![_0KPM_6}VZ_8`731}`@%CA7](https://github.com/community/community/assets/170230394/ad6583f9-475e-4e94-a2b8-0d98e10fac6e)
![C)J@M}HB0{Y0XL142Q(2`ZG](https://github.com/community/community/assets/170230394/40a6f414-68c3-4829-8c26-3fc1472cb1a3)

**Comment by ebndev** [comment]
Hi @nebulas16,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this game. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
